app/services/proofing/lexis_nexis/ddp/response_redacter.rb

```diff
@@ -109,6 +164,9 @@ class ResponseRedacter
           session_id
           session_id_query_count
           summary_risk_score
+          time_zone
```

Comment:
This just made me Google "least populated time zones" to see if there would be a way to identify an individual by their time zone. (Answer: [no](https://earthscience.stackexchange.com/questions/12504/least-populated-time-zones))